spec/features/recipes/recipes_page_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'user visits recipes page' do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
+    visit('/')
+  end
+
+  it 'shows user an example data' do
+    expect(page).to have_content Recipe.first.name
```

Comment:
Please use strings in your test code for better comprehension. I don't really know what is under `Recipe.first.name`. It should look like this:

```ruby
expect(page).to have_content('Scrambled Eggs')
```